src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -27,37 +27,37 @@ Proof-of-stake comes with a number of improvements to the proof-of-work system:
+Proof-of-stake is the underlying mechanism that activates validators upon receipt of enough stake. For Ethereum, users will need to stake 32 ETH to become a validator. Validators are chosen at random to create blocks and are responsible for checking and confirming blocks they don't create. A user's stake is also used as a way to incentivise good validator behaviour. For example, a user can lose a portion of their stake for things like going offline (failing to validate) or their entire stake for deliberate collusion.
 
 ## How does Ethereum's proof-of-stake work? {#how-does-pos-work}
 
-Unlike proof-of-work, validators don't need to use significant amounts of computational power because they're selected at random and aren't competing. They don't need to mine blocks, they just need to create blocks when chosen and validate proposed blocks when they're not. This validation is known as attesting. You can think of attesting as saying "this block looks good to me". Validators get rewards for proposing new blocks and for attesting to ones they've seen.
+Unlike proof-of-work, validators don't need to use significant amounts of computational power because they're selected at random and aren't competing. They don't need to mine blocks; they just need to create blocks when chosen and validate proposed blocks when they're not. This validation is known as attesting. You can think of attesting as saying "this block looks good to me." Validators get rewards for proposing new blocks and for attesting to ones they've seen.
 
 If you attest to malicious blocks, you lose your stake.
 
 ### The beacon chain {#the-beacon-chain}
 
 When Ethereum replaces proof-of-work with proof-of-stake, there will be the added complexity of [shard chains](/eth2/shard-chains/). These are separate blockchains that will need validators to process transactions and create new blocks. The plan is to have 64 shard chains and they all need a shared understanding of the state of the network. So extra coordination is needed and this will be done by [the beacon chain](/eth2/beacon-chain/).
 
-The beacon chain receives state information from shards and makes it available for other shards so that the network can stay in sync. The beacon chain will also manage the validators, from registering their stake deposits to issuing their rewards and penalties.
+The beacon chain receives state information from shards and makes it available for other shards so that the network can stay in sync. From registering their stake deposits to issuing their rewards and penalties, the beacon chain will also manage the validators.
```

Comment:
Only change I'm not so sure on... feel the sentence structure just reads a little more awkwardly after flipping, could be convinced though, thoughts?

Reply:
@wackerow 
For sure! This weekend, I'll look through the rest of the pages for more potential edits to help out with the content. I see what you're referring to with ***The beacon chain*** section's flow, so I just reverted the original sentence in my most recent commit. I also added a few more edits that I noticed when rereading the doc.